app/src/main/java/com/example/farmeraid/data/model/MarketModel.kt

```diff
@@ -5,7 +5,8 @@ import com.example.farmeraid.data.QuotasRepository
 class MarketModel {
     data class Market(
         val id : String,
-        val name: String
+        val name: String,
+        val quotaID: String
```

Comment:
Why are we adding quotaID back? I thought we don't need the quotaID anymore and the quota for a market is just referenced by market ID

Reply:
Right, I totally forgot we discussed that. I will change it and reseed db to follow this pattern